migrating_from_ocp_3_to_4/installing-3-4.adoc

```diff
@@ -0,0 +1,63 @@
+[id='installing-3-4']
+= Installing the Migration Toolkit for Containers
+include::modules/common-attributes.adoc[]
+:context: installing-3-4
+
+toc::[]
+
+You can install the {mtc-full} ({mtc-short}) on {product-title} 3 and on {product-title} {product-version} clusters.
+
+[IMPORTANT]
+====
+You must install the same {mtc-short} version on all clusters.
+====
+
+By default, the {mtc-short} web console and the `Migration Controller` pod run on the target cluster.
+
+You can configure the `Migration Controller` custom resource manifest to run the {mtc-short} web console and the `Migration Controller` pod on a link:https://access.redhat.com/articles/5064151[source cluster or on a remote cluster].
+
+== Installing the {mtc-short} Operator in a restricted network environment
```

Comment:
Have you considered making this its own assembly? And having a regular installing assembly and an installing in a restricted environment assembly?

I like how you separated out Upgrading into its own assembly, and I'm thinking user story-wise it makes sense that users would only be doing installation way or the other.

Looking back at the additional resources section below and what I commented on, it looks like more of just an FYI on these sections, and not instructions to actually do them.

Reply:
Actually, I did have separate sections for connected and restricted network environments within a single assembly. It resulted in a lot of repetition. That's why I decided to use a single assembly for both installations. The installation procedures are fairly similar. The biggest difference is that the user has to create a local Operator catalog in order to install the MTC operator. Perhaps the "Additional resources" header is misleading. The xrefs are for creating the local catalog.

Maybe I could break them out into separate assemblies.